code/datums/diseases/advance/symptoms/beesymptom.dm

```diff
@@ -0,0 +1,59 @@
+/datum/symptom/beesease
+	name = "Bee Infestation"
+	desc = "Causes the host to cough toxin bees and occasionally synthesize toxin."
+	stealth = -2
+	resistance = 2
+	stage_speed = 1
+	transmittable = 1
+	level = 9
+	severity = 1
+	symptom_delay_min = 15
+	symptom_delay_max = 30
+	var/honey = FALSE
+	var/infected_bees = FALSE
+	threshold_desc = "<b>Resistance 14:</b> Host synthesizes honey instead of toxins, bees now sting with honey instead of toxin.<br>\
+					  <b>Transmission 10:</b> Bees now contain a small amount of infected blood"				
+
+/datum/symptom/beesease/Start(datum/disease/advance/A)
+	if(!..())
+		return
+	if(A.properties["resistance"] >= 14)
+		honey = TRUE
+	if(A.properties["transmission"] >= 10)
+		infected_bees = TRUE
+
+/datum/symptom/beesease/Activate(datum/disease/advance/A)
+	if(!..())
+		return
+	var/mob/living/M = A.affected_mob
+	switch(A.stage)
+		if(2)
+			if(prob(2))
+				to_chat(M, "<span class='notice'>You taste honey in your mouth.</span>")
+		if(3)
+			if(prob(10))
+				to_chat(M, "<span class='notice'>Your stomach rumbles.</span>")
+			if(prob(5))
+				if(honey)
+					to_chat(M, "<span class='notice'>You taste even more honey.</span>")
+					M.reagents.add_reagent(/datum/reagent/consumable/honey, 2)
+				else
+					to_chat(M, "<span class='danger'>Your stomach stings painfully.</span>")
```

Comment:
Why is this outside the prob?

Reply:
that's how it is in the original beesease disease (not symptom)